web/api/README.md

```diff
@@ -0,0 +1,44 @@
+# CodeChecker server Thrift API
+
+This directory contains the API IDL files and the generated API stubs for CodeChecker.
+[Apache Thrift](https://thrift.apache.org/) is used to generate the stubs
+for various programming languages.
+
+The Thrift compiler is executed inside a [Docker](https://www.docker.com/)
+container so docker needs to be installed to generate the stubs.
+
+## API change workflow:
+
+Assuming the current api version is 6.24.0 and no breaking change was
+introduced.
+
+### 1. Modify the Thrift files.
+
+Update the versions in the setup.py files to 6.24.0.dev1.
+NPM de package should be pushed with a dev tag.
+
+Run the command `make build` to generate the stubs.
+
+Publish dev versions of the api:
+ - In the setup.py used 6.24.0.dev1 until version until the api is accepted.
+ - Use v6.24.0 version in the package.json publish with a tag until API is tested:
+   `npm publish --tag dev1`
+
+### 2. Server/client implementation for the API change
+
+Update the requirements.txt and extend the server with the new
```

Comment:
Please use [list](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lists) here and highlight important things.